docs/configuration.md

```diff
@@ -156,6 +156,13 @@ of the most common options to set are:
   </td>
 </tr>
 <tr>
+ <td><code>spark.executor.instances</code></td>
```

Comment:
I think this is actually YARN-specific right now? At least it is according to `SparkSubmit.scala`